front/app/components/EventCard/DateBlocks/styling.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { colors, media } from 'utils/styleUtils';
+
+export const EventDateBlockWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+
+  &.second {
+    margin-top: 14px;
+  }
+`;
+
+export const EventDateBlockLabel = styled.div`
+  color: ${colors.label};
+  font-size: 12px;
+  line-height: normal;
+  font-weight: 600;
+  text-transform: uppercase;
+  margin-bottom: 4px;
+`;
+
+export const EventDateBlock = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+`;
+
+export const EventDate = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: stretch;
+  padding: 6px;
+  border-radius: ${(props: any) => props.theme.borderRadius};
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  background: #f5f6f7;
+  border: solid 1px #ccc;
+  border-bottom: none;
+
+  ${media.smallerThanMinTablet`
+    padding: 4px;
+  `}
+`;
+
+export const EventMonth = styled.div`
```

Comment:
Do you mean that I should just convert all `div`s here to `time` elements? Or is it more about the `datetime` attribute? Because for the latter, I am not sure on which element(s) I would have to put it in order to store the starting date AND the duration...

Reply:
My honest response: I haven't done this before, but think we can be more semantic here. Would also have to research what best practices are. So I'd just Google for it. :-)